services/frontend/pkg/revaconfig/config.go

```diff
@@ -63,6 +63,11 @@ func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error
 		}
 	}
 
+	readOnlyUserAttributes := []string{}
+	if cfg.ReadOnlyUserAttributes != nil {
+		readOnlyUserAttributes = cfg.ReadOnlyUserAttributes
+	}
```

Comment:
Is this necessary? Can't we just assign `cfg.ReadOnlyUserAttributes` directly? It shouldn't care if it is `nil`

Reply:
I added that because I thought it would be nicer for the frontend people if they'd just have a list to iterate over and not have to check for nil. I'm totally fine with both.

Reply:
Fine with me :+1: